### DISCUSSION TITLE
New logger() type - success

### DISCUSSION THREAD

**Discussion by Josh-Dovey** [laravel/framework#52018] [Ideas]
I apologise in advance if this idea has been raised in the past. I have tried searching for a similar feature request and have not found anything.

When using logger() we have various logger types which make filtering log files with log viewer packages much easier. We have the following log types emergency, alert, critical, error, warning, notice, info and debug. Could another type be added?

Can a success log type be added? 

Is there something in place that already mimics this? There is an error type. So for me, having a success type would make filtering these logs even easier.

Thanks in advance for your help, appreciate all the work put into this framework. It's fantastic!

**Comment by ghost** [comment]
"Is there something in place that already mimics this" : info

**Reply by Josh-Dovey** [reply]
I use info when logging different stages of a process. I use error when catching an exception and so on. The idea was after a process has completed and is successful, a success log could be used.

**Reply by ghost** [reply]
These are the monolog log types

    protected static $levels = [
        self::DEBUG     => 'DEBUG',
        self::INFO      => 'INFO',
        self::NOTICE    => 'NOTICE',
        self::WARNING   => 'WARNING',
        self::ERROR     => 'ERROR',
        self::CRITICAL  => 'CRITICAL',
        self::ALERT     => 'ALERT',
        self::EMERGENCY => 'EMERGENCY',
    ];

Maybe warning could be used as succeeded.
